modules/ROOT/pages/_partials/secrets-manager.adoc

```diff
@@ -0,0 +1,715 @@
+|xref:anypoint-cli::secrets-manager.adoc#secret-truststore-create[secrets-mgr:truststore:create]| Creates a new truststore secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-truststore-describe[secrets-mgr:truststore:describe]| Shows details of a truststore secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-truststore-list[secrets-mgr:truststore:list]| Lists all truststore secrets in a secret group
+|xref:anypoint-cli::secrets-manager.adoc#secret-truststore-modify[secrets-mgr:truststore:modify]| Modifies a truststore secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-truststore-replace[secrets-mgr:truststore:replace]| Replaces an existing truststore secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-context-create[secrets-mgr:tls-context:mule:create]| Creates a new Mule TLS Context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-context-describe[secrets-mgr:tls-context:mule:describe]| Shows details of a Mule TLS context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-context-list[secrets-mgr:tls-context:mule:list]| Lists all Mule TLS context secrets in a secret group
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-context-modify[secrets-mgr:tls-context:mule:modify]| Modifies a Mule TLS context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-context-replace[secrets-mgr:tls-context:mule:replace]| Replaces an existing Mule TLS context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-flex-create[secrets-mgr:tls-context:flex-gateway:create]| Creates a new Flex Gateway TLS Context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-flex-describe[secrets-mgr:tls-context:flex-gateway:describe]| Shows details of a Flex Gateway TLS context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-flex-list[secrets-mgr:tls-context:flex-gateway:list]| Lists all Flex Gateway TLS context secrets in a secret group
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-flex-modify[secrets-mgr:tls-context:flex-gateway:modify]| Modifies a Flex Gateway TLS context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-flex-replace[secrets-mgr:tls-context:flex-gatway:replace]| Replaces an existing Flex Gateway TLS context secret
+
+
+|===
+
+// end::summary[]
+
+// tag::commands[]
+
+[[secret-group-create]]
+== secrets-mgr:secret-group:create
+
+----
+> secrets-mgr:secret-group:create [flags] 
+----
+
+Creates a new secret group with the name specified in `--name`.
+
+Prompt the `--downloadable` flag if the secrets in this group are referenced in an API Manager proxy.
+
+This command also accepts the default `--help` flag.
+
+
+[[secret-group-delete]]
+== secrets-mgr:secret-group:delete
+
+----
+> secrets-mgr:secret-group:delete [flags] 
+----
+Deletes the secret group specified in `--id` +
+
+[WARNING]
+This command does not prompt twice before deleting. If you send a delete instruction, it does not ask for confirmation.
```

Comment:
This is confusing - "twice" implies two confirmations. Also, it's not clear to me what "it" refers to in the second sentence (delete instruction or...?"

What about "This command doesn't prompt for confirmation before deleting."